### ISSUE TITLE
Header Adjustments

### ISSUE THREAD

**Issue by DeltaDizzy** [DeltaDizzy/Kop-Wiki-Backup#8]
- [ ] Increase header width
- [x] Exclude Home page from nav bar
- [ ] Figure out document.referrer OR
- [ ] Algorithm to generate a link to the upper-level page
  - [ ] Re-organize file directory

**Comment by democat3457** [comment]
:ok_hand: Sounds good!

**Comment by DeltaDizzy** [comment]
Seems like you have the second item taken care of.

**Comment by democat3457** [comment]
Yep. Now I just have to figure out how to add a link to the upper-level page.
